<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<HTML>
 <HEAD>
   <TITLE> [rabbitmq-discuss] Autorecovery Issue
   </TITLE>
   <LINK REL="Index" HREF="index.html" >
   <LINK REL="made" HREF="mailto:rabbitmq-discuss%40lists.rabbitmq.com?Subject=Re%3A%20%5Brabbitmq-discuss%5D%20Autorecovery%20Issue&In-Reply-To=%3CetPan.538f0502.6b68079a.e723%40mercurio.local%3E">
   <META NAME="robots" CONTENT="index,nofollow">
   <style type="text/css">
       pre {
           white-space: pre-wrap;       /* css-2.1, curent FF, Opera, Safari */
           }
   </style>
   <META http-equiv="Content-Type" content="text/html; charset=us-ascii">
   <LINK REL="Previous"  HREF="036447.html">
   <LINK REL="Next"  HREF="036452.html">
 </HEAD>
 <BODY BGCOLOR="#ffffff">
   <H1>[rabbitmq-discuss] Autorecovery Issue</H1>
    <B>Michael Klishin</B> 
    <A HREF="mailto:rabbitmq-discuss%40lists.rabbitmq.com?Subject=Re%3A%20%5Brabbitmq-discuss%5D%20Autorecovery%20Issue&In-Reply-To=%3CetPan.538f0502.6b68079a.e723%40mercurio.local%3E"
       TITLE="[rabbitmq-discuss] Autorecovery Issue">mklishin at gopivotal.com
       </A><BR>
    <I>Wed Jun  4 12:37:09 BST 2014</I>
    <P><UL>
        <LI>Previous message: <A HREF="036447.html">[rabbitmq-discuss] Autorecovery Issue
</A></li>
        <LI>Next message: <A HREF="036452.html">[rabbitmq-discuss] Autorecovery Issue
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#36449">[ date ]</a>
              <a href="thread.html#36449">[ thread ]</a>
              <a href="subject.html#36449">[ subject ]</a>
              <a href="author.html#36449">[ author ]</a>
         </LI>
       </UL>
    <HR>  
<!--beginarticle-->
<PRE>On 4 June 2014 at 15:22:30, Ranjitha Shet (<A HREF="https://lists.rabbitmq.com/cgi-bin/mailman/listinfo/rabbitmq-discuss">ranjithavshet09 at gmail.com</A>) wrote:
&gt;<i> &gt; Caught an exception during connection recovery!
</I>&gt;<i> java.net.NoRouteToHostException: No route to host: connect  
</I>
This means that RabbitMQ Java client cannot connect to the broker because
OS cannot route packets to it (expected with the network cable being pulled out).

There is a bug fix in the upcoming 3.3.2 release that makes sure any exception
(including the one above) results in a reconnection attempt after a grace period.
Previously only some network I/O exceptions triggered reconnection.

Try with a nighly build of the client if you can.&#160;
--  
MK  

Software Engineer, Pivotal/RabbitMQ
</PRE>




























<!--endarticle-->
    <HR>
    <P><UL>
        <!--threads-->
	<LI>Previous message: <A HREF="036447.html">[rabbitmq-discuss] Autorecovery Issue
</A></li>
	<LI>Next message: <A HREF="036452.html">[rabbitmq-discuss] Autorecovery Issue
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#36449">[ date ]</a>
              <a href="thread.html#36449">[ thread ]</a>
              <a href="subject.html#36449">[ subject ]</a>
              <a href="author.html#36449">[ author ]</a>
         </LI>
       </UL>

<hr>
<a href="https://lists.rabbitmq.com/cgi-bin/mailman/listinfo/rabbitmq-discuss">More information about the rabbitmq-discuss
mailing list</a><br>
</body></html>
